crates/stc_ts_type_checker/tests/tsc-stats.rust-debug

```diff
@@ -1,5 +1,5 @@
 Stats {
-    required_error: 4161,
-    matched_error: 5534,
-    extra_error: 983,
+    required_error: 4150,
+    matched_error: 5545,
+    extra_error: 1012,
```

Comment:
@kdy1 I'm not sure what `required_error` error mean relative to `matched_error`. 

So far I've understood that (correct me if im wrong):
- `extra_error`s are the ones that we should not be emiting but we do. This should go down.
- `matched_error`s are the ones that matched expected errors. This still counts if there are `extra_error`s in a test case (?). This should go up.

Reply:
`required_error` means we have to emit those error for 100% parity with `tsc`

Reply:
But well, I think we should document it somewhere, or change the name

Reply:
you should rename it to `missing_error`